koku/masu/config.py

```diff
@@ -74,6 +75,9 @@ class Config:
     # Limit the number of enabled tags:
     ENABLED_TAG_LIMIT = ENVIRONMENT.int("TAG_ENABLED_LIMIT", default=DEFAULT_ENABLED_TAG_LIMIT)
 
+    # Wait period for the the delay summary queue.
+    DELAYED_RESUMMARY_TIME = ENVIRONMENT.int("DELAYED_RESUMMARY_TIME", default=DEFAULT_ENABLED_TAG_LIMIT)
```

Comment:
I understand why this was put into this file, but I think we should stop adding new things to this file and just rely on the settings file. It can be a bit confusing to track config variables thruout our code because they come from different places.

All of our celery config lives in [settings.py](https://github.com/project-koku/koku/blob/cb49aa39c0bf96706e458cbfea29c705a51c1d33/koku/koku/settings.py#L552-L565) anyway, so it would make sense to put this there too.

I would move this to koku.settings, and then use it wherever it is needed like: 
```
settings.py:
DELAYED_RESUMMARY_TIME = ENVIRONMENT.int("DELAYED_RESUMMARY_TIME", default=3600)

where used:
from django.conf import settings
settings.DELAYED_RESUMMARY_TIME
```